test/e2e-upgrade-tests.sh

```diff
@@ -36,8 +36,48 @@ source $(dirname $0)/e2e-common.sh
 OPERATOR_DIR=$(dirname $0)/..
 KNATIVE_SERVING_DIR=${OPERATOR_DIR}/..
 
-function knative_setup() {
+function install_latest_operator_release() {
+  header "Installing Knative Serving operator latest public release"
+  local url="https://github.com/knative/serving-operator/releases/download/${LATEST_SERVING_RELEASE_VERSION}"
```

Comment:
any reason not to combine line 41 and 42 together?

local full_url="https://github.com/knative/serving-operator/releases/download/${LATEST_SERVING_RELEASE_VERSION}/serving-operator.yaml"

Reply:
No, just legacy.